packages/website/src/theme/SearchBar/SearchButton.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styles from './SearchButton.module.css';
+
+export default function SearchButton({ onClick }) {
```

Comment:
Do we need to move `SearchButton` component from the Docusaurus theme to the DocSearch-React package? 
Probably it makes more sense to have it inside DocSearch package? I'm not sure and I didn't have time to move it there.

Reply:
For now it's fine, I think we'll take a decision later. To support lazy loading even though we ship the button in DocSearch, we'll probably be able to expose the button in `docsearch-react/input`. Wdyt?

Reply:
Ah, right. We'll later try it that way.